src/main/java/me/tomsdevsn/hetznercloud/HetznerCloudAPI.java

```diff
@@ -2381,22 +2190,62 @@ public PlacementGroupResponse createPlacementGroup(PlacementGroupRequest placeme
      * @return ActionResponse
      */
     public String deletePlacementGroup(long id) {
-        return restTemplate.exchange(
+        return delete(
                 API_URL + "/placement_groups/" + id,
-                HttpMethod.DELETE,
-                httpEntity,
-                String.class).getBody();
+                String.class);
     }
 
-    /**
-     * Converts a Date to the ISO-8601 format
-     *
-     * @param date Date to be converted
-     * @return Date in ISO-8601 format
-     */
-    public String convertToISO8601(Date date) {
```

Comment:
Is there any reason why you've removed this method? I would more like to deprecate it first and remove it with a next release if you think it shouldn't be here.

Reply:
It was unused, also the test method for it was empty. If its part of the public API I will re-add it.

Reply:
The initial test suite was written by someone else, I didn't find any time to finish the tests.

And yes, it was part of the Public API. I would please you to add it back.

Reply:
I re-added the method